controllers/volume/volume_controller.go

```diff
@@ -892,10 +923,61 @@ func sanitizeCNSErrorMessage(msg string) string {
 func attachmentToVolumeStatus(
 	volumeName string,
 	attachment cnsv1alpha1.CnsNodeVmAttachment) vmopv1.VirtualMachineVolumeStatus {
+
 	return vmopv1.VirtualMachineVolumeStatus{
 		Name:     volumeName, // Name of the volume as in the Spec
 		Attached: attachment.Status.Attached,
 		DiskUUID: attachment.Status.AttachmentMetadata[AttributeFirstClassDiskUUID],
 		Error:    sanitizeCNSErrorMessage(attachment.Status.Error),
+		Type:     vmopv1.VirtualMachineStorageDiskTypeManaged,
 	}
 }
+
+func updateVolumeStatusWithLimit(
+	ctx *pkgctx.VolumeContext,
+	c client.Reader,
+	status *vmopv1.VirtualMachineVolumeStatus) error {
+
+	// Find the name of the PVC.
+	var pvcName string
```

Comment:
The only caller of this already has the `volume.PersistentVolumeClaim.ClaimName` handy just pass it in here?